frontend/src/pull-card/index.tsx

```diff
@@ -88,6 +88,49 @@ function PullCard({pull, show}: {pull: Pull, show: boolean}) {
    );
 });
 
+export const ClosedPullCard = memo(
+function ClosedPullCard({pull, show}: {pull: Pull, show: boolean}) {
+   const wasMerged = !!pull.merged_at;
+   return (
+      <Card display={show ? undefined : "none"}>
+         <RefreshButton pull={pull}/>
+         <Box>
+            <Box mt={-2} ml={-2} mb={2}>
+               <chakra.span pr={2}>
+                  {wasMerged ?
+                     <FontAwesomeIcon icon={faCodeMerge} size="lg" color="var(--pull-merged)"/> :
+                     <FontAwesomeIcon icon={faXmark} color="var(--pull-closed)"/>
+                  }
+               </chakra.span>
+               <chakra.span color="var(--pull-title)">
+                  {formatDate(pull.closed_at)}
```

Comment:
Im assuming a non null `merged_at` is always = to `closed_at`. Is that right?

Reply:
I presume that for merged pulls, `merged_at === closed_at`.  The docs don't explicitly say, but I can't imagine what else it'd be.